src/v3/src/components/Form/Form.tsx

```diff
@@ -47,13 +47,14 @@ const Form: FunctionComponent<{
   const handleSubmit = useCallback(async (e: SubmitEvent) => {
     e.preventDefault();
     setWidgetRendered(false);
-    setMessage(undefined);
 
-    // TODO: Can remove after OKTA-586475 is resolved
+    // TODO: Can remove after OKTA-657627 is resolved
     if (!dataSchemaRef?.current?.submit) {
       return;
     }
 
+    setMessage(undefined);
```

Comment:
Before, the info-box message that I trigger when clicking the stepper button was getting set as `undefined` and never appearing, so I had to move it below the short-circuit if-statement above so that it will render and my own `setMessage(undefined)` call in the `nextStepIndex` function handles clearing the message